.travis.yml

```diff
@@ -69,9 +69,12 @@ matrix:
 before_install:
   - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
   - git submodule update --init --recursive
+  - 'curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"'
```

Comment:
The old way could be working again, but that's not important.

Reply:
Agreed, plan is to remove Travis in the near future anyway.